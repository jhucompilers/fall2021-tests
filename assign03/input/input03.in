PROGRAM input03;
  VAR sum, n: INTEGER;

BEGIN
  -- read integer from user
  READ n;
  -- continue adding positive numbers
  sum := 0;
  WHILE n > 0 DO
    sum := sum + n;
    READ n;
  END;
  -- output the sum
  WRITE sum;
END.
