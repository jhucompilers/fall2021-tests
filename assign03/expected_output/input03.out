0,VAR,sum,INTEGER
0,VAR,n,INTEGER
